Vision/src/Connection/Rest.php

```diff
@@ -27,8 +27,7 @@
  * Implementation of the
  * [Google Cloud Vision JSON API](https://cloud.google.com/vision/reference/rest/).
  *
- * @deprecated This class is no longer supported and will be removed in a future
- * release.
+ * @internal
```

Comment:
For now I have changed this, but since this was already deprecated, should I leave it as is?